code/go/internal/validator/spec.go

```diff
@@ -202,15 +202,15 @@ func (s Spec) rules(pkgType string, rootSpec spectypes.ItemSpec) validationRules
 		{fn: semantic.ValidateDateFields, types: []string{"integration", "input"}},
 		{fn: semantic.ValidateRequiredFields, types: []string{"integration", "input"}},
 		{fn: semantic.ValidateExternalFieldsWithDevFolder, types: []string{"integration", "input"}},
-		{fn: semantic.WarnOn(semantic.ValidateVisualizationsUsedByValue), types: []string{"integration"}, until: semver.MustParse("3.0.0")},
-		{fn: semantic.ValidateVisualizationsUsedByValue, types: []string{"integration"}, since: semver.MustParse("3.0.0")},
+		{fn: semantic.WarnOn(semantic.ValidateVisualizationsUsedByValue), types: []string{"integration", "content"}, until: semver.MustParse("3.0.0")},
+		{fn: semantic.ValidateVisualizationsUsedByValue, types: []string{"integration", "content"}, since: semver.MustParse("3.0.0")},
 		{fn: semantic.ValidateILMPolicyPresent, since: semver.MustParse("2.0.0"), types: []string{"integration"}},
 		{fn: semantic.ValidateProfilingNonGA, types: []string{"integration"}},
-		{fn: semantic.ValidateKibanaObjectIDs, types: []string{"integration"}},
+		{fn: semantic.ValidateKibanaObjectIDs, types: []string{"integration", "content"}},
 		{fn: semantic.ValidateRoutingRulesAndDataset, types: []string{"integration"}, since: semver.MustParse("2.9.0")},
 		{fn: semantic.ValidateKibanaNoDanglingObjectIDs, since: semver.MustParse("3.0.0")},
 		{fn: semantic.ValidateKibanaFilterPresent, since: semver.MustParse("3.0.0")},
-		{fn: semantic.ValidateKibanaNoLegacyVisualizations, types: []string{"integration"}, since: semver.MustParse("3.0.0")},
+		{fn: semantic.ValidateKibanaNoLegacyVisualizations, types: []string{"integration", "content"}, since: semver.MustParse("3.0.0")},
```

Comment:
Should these changes be mentioned in the changelog too?

Reply:
Added a new changelog entry for these changes in https://github.com/elastic/package-spec/pull/884/commits/dd3313942569019bbb54490d96c5d2a42fdc7799